frontend/src/utils/formatters.ts

```diff
@@ -35,6 +35,19 @@ export function formatNumber(n: number) {
   }
 }
 
+/**
+ * Formats numbers with 'K' for thousands, 'M' for millions for axis labels.
+ */
+export function formatAxisNumber(n: number) {
```

Comment:
This function could have a more general name maybe?

Reply:
fixed